bedrock/base/templates/base-pebbles.html

```diff
@@ -94,6 +94,8 @@
       {% include 'includes/protocol/navigation/menu-mozilla/index.html' %}
     {% endblock %}
 
+    {% block page_banner %}{% endblock %}
```

Comment:
Can you explain a bit about why you feel it should come above the navigation? This was actually the first thing I thought to do here, as I always felt like it was out of place above and made the nav underneath a potentially confusing UX. I feel making it appear below the nav fits in better with the hierarchy of the page, and more like a permanent feature (vs feeling like an injected ad?). I’d be interested in opinions from others here too.

Reply:
Well to be honest I've always thought of this as an injected ad and not part of the page anyway. I was just noting that it's currently above the nav so moving it below is a change, but maybe it's a change for the better. We should see with @dzingeek and @justinkropp think.

Reply:
@dzingeek @justinkropp to help give a little more context to this conversation, we're talking about making our current system for banners more reusable as a feature on the site. Right now the only banner we have is the homepage fundraising banner, but in the future we could show other things, such as a promo for Fenix as an example.

### Currently, the banner appears above the main site navigation like so:

![image](https://user-images.githubusercontent.com/400117/75972727-d26dba80-5ecb-11ea-8fad-e91387cc6f93.png)

### With this change, I'm proposing we move it below the nav:

![image](https://user-images.githubusercontent.com/400117/75972809-f3cea680-5ecb-11ea-972f-fe56e97f866e.png)

My thought here is that banners will fits a bit better with the general page hierarchy of the site when they appear below the navigation, but I'm open to keeping it as-is if others disagree.

Reply:
@alexgibson Thanks for tackling this and for asking for feedback. I'm on @craigcook's side here in thinking that the banner makes more sense to me -- from an overall user experience perspective -- living above the primary navigation. From some preliminary research, similar fundraising banners typically live at the very top of the page above the nav. Placing the banner below the nav will, imo, give the banner primary importance and potentially confuse the user into thinking that it is permanent page content. 

I really like this project and would like to see the design of this banner evolve a bit and get themed: one theme for the normal mozilla fundraising banner, and another theme for more generic Firefox promotional messaging.

Reply:
Ok, I'll leave it as-is for now - although we should consider this is not just a fundraising banner anymore. It may evolve to have multiple uses. 

Personally I question whether the current positioning is somewhat confusing, as users new to the site might typically expect navigation to appear at the top of the page, and scrolling down can then trigger opening the menu panels unexpectedly.

Reply:
I also agree with @craigcook and @justinkropp however I do get @alexgibson concerns. I think this might be a good candidate for some user testing to see how people respond to it from different placements. Antoher great point from @alexgibson is that this is no longer just a fundraiser banner and could have multiple uses. So my concern there is how often will this be used? If it's on the page more than it's not than we shoudl re-evalute it's placement a bit more.

Reply:
> So my concern there is how often will this be used? If it's on the page more than it's not than we shoudl re-evalute it's placement a bit more.

Yeah, I think for now this is fine to leave as-is. We're not (yet) building any other types of banner. But it may be worth re-evaluation when we make further use of the feature.

Thanks for the feedback all!